### PR TITLE
Jleslie/ocp 311 4 howto updates

### PR DESCRIPTION
**Pull request by jleslie85**
Tom, Ross
Please review README page updates to include install and deployment path for either OCP 3.11 and 4
Thanks,
Joe

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,20 +62,41 @@ sudo chown -R root:root /mnt/local-storage
...
 
-### Create the nuodb project (if not already created)
+### Create the "nuodb" project (if not already created)
 
 &ensp; `kubectl new-project nuodb`
 
-### Create the Kubernetes container image pull secret
+### Create the Kubernetes image pull secret to access the Red Hat Container Catalog (RHCC)
+
+This secret will be used to pull the NuoDB Operator and NuoDB container images
```
Comment (thebithead) [inline README.md]: I think you should say that the credentials for that secret are your RedHat Login, and you must create a RedHat account in order to pull from the Red Hat Container Catalog.

File: README.md [hunk 2]
```diff
@@ -90,7 +123,7 @@ To deploy the NuoDB database into your Kubernetes cluster, run the following com
 kubectl create -n $OPERATOR_NAMESPACE -f deploy/cr.yaml
  ```
 
-&ensp; &ensp; _**Note:** Before running the above create Customer Resource file command, this is where you have the option to configure your database just the way you want it._
+_**Note:** Before running the above create Customer Resource file command, this is where you have the option to configure your database just the way you want it._
```
Comment (thebithead) [inline README.md]: It should say "Custom Resource" not "Customer Resource".

File: README.md (hunk 1 above)
Comment (jleslie85) [reply README.md]: For this section , "Create the Kubernetes image pull secret to access the Red Hat Container Catalog (RHCC)" I clarified they must enter their Red Hat login credentials.

File: README.md (hunk 2 above)
Comment (jleslie85) [reply README.md]: Thanks for locating this. I found the same in the video overlay text also on Friday. Didn't realize it was here also. (Fixed).

**Comment by thebithead** [comment]
Looks good.  I don't know if you need the docker secret when using ocp4 and "subscribing" to the NuoDB operator.  I will merge this into master.

**Pull request merged**